_posts/2020-12-03-How-Create-a-New-index.html-file.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: How to Create a New index.html file
+description: 
+summary: 
+tags: 
+---
+We had trouble creating a blank index.html file, and we would forget to add the basic elements of an html page. This post is to avoid repeating the same mistake of being unsure in creating an html page. Open Up terminal and start with the initial commands up until opening Projects up.
+Once in projects, commit the following commands to create an empty index.html file:
+
+* mkdir Thinkment - mkdir allows the user to create a folder with the name they chose to. name it
+
+* cd Thinkment - Once you have made the file, open up the file.
+
+* touch index.html - This adds an empty html page to start working on your page.
+
+* touch README.md - Contains other files in the directory, and is a form of documentation.
+
+* touch script.js - creates an empty file titled script.js
+
+* touch style.css - Allows you to style your page or website
+
+After completing the coding structure in Terminal, opne up Sublime text and open up the folder you created. On sublime text in the index.html section, add the skeleton from the following website.
+`<!DOCTYPE html>`
```

Comment:
instead of putting back ticks around every element separately, put three before the code block and three after. that will maintain new lines.
```
<!DOCTYPE html>
<html>
    <head>
        <!-- head definitions go here -->
    </head>
    <body>
        <!-- the content goes here -->
    </body>
</html>
```

Reply:
![Uploading Screen Shot 2020-12-03 at 12.49.55 PM.png…]()